src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerApp.java

```diff
@@ -167,6 +167,7 @@ private static void handle(PAThreadException exception) {
     // what to do when the thread has stopped due to an unexpected exception.
     LOG.error("Thread: {} ran into an uncaught exception: {}", exception.getPaThreadName(),
         exception.getInnerThrowable());
+    exception.getInnerThrowable().printStackTrace();
```

Comment:
Should we do this:
LOG.error("Thread: {} ran into an uncaught exception: {}", exception.getPaThreadName(),
        exception.getInnerThrowable(), exception);

to capture the stack trace in the logs ?

Reply:
I'll incorporate this.